Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Menu/MenuPage.xaml

```diff
@@ -26,21 +28,22 @@
                        TooltipPlacement="{Binding TooltipPlacement.Value, Mode=TwoWay}">
 
             <controls:MenuItem x:Name="FileMenu"
-                               controls:Menu.InputGestureText="Alt+F">
+                               controls:Menu.InputGestureText="Alt+F"
+                               Header="^File">
```

Comment:
you can add any template to the header, maybe a single Image or a stack panel or even a grid how can I know which element containing the text without work around :)
If it is requested then we could include an icon property in the menuItem or provide a way to access the required text element.